programs/delayed_node/main.cpp

```diff
@@ -160,26 +167,24 @@ int main(int argc, char** argv) {
          elog("Error parsing configuration file: ${e}", ("e", e.what()));
          return 1;
       }
-      if( !options.count("plugins") )
```

Comment:
You should keep the "plugins" option. We don't want to change the behaviour of the executable.

Reply:
this is done https://github.com/bitshares/bitshares-core/commit/1326a5c126d8fcf99cfa717925dac58c0bbb0a0d